src/clr-angular/data/datagrid/datagrid-row.spec.ts

```diff
@@ -246,25 +271,22 @@ export default function(): void {
                 expect(context.clarityElement.querySelector(".spinner")).not.toBeNull();
             });
 
-            it("doesn't display the details when collapsed", function() {
-                expect(context.clarityElement.textContent).toMatch("Hello world");
```

Comment:
Why delete these `expect`s? Looks like they should still work, and they test that we still display the original content of the row. Or are we just not rendering anything in these tests? If that's the case, then the test doesn't actually mean anything since "always displaying nothing" clearly satisfies "does not display the details".

Reply:
I'm not sure what I was thinking. I'll put it back and updated the test component.